randovania/games/dread/json_data/Burenia.txt

```diff
@@ -2153,7 +2193,9 @@ Extra - asset_id: collision_camera_021
               Use Spin Boost
               All of the following:
                   Gravity Suit
-                  Flash Shift or Simple IBJ
+                  Any of the following:
+                      Flash Shift or Simple IBJ
+                      Speed Booster and Disabled Door Lock Randomizer
```

Comment:
As i stated in the Discord, placing an event in the upper part of Gravity Suit Room would also be perfectly safe to require here as well. "Disabled Door Lock Randomizer or Gravity Suit Shinespark"

Reply:
What does placing an event in the upper part of gravity suit room do for your ability to get across the bottom, into the recharge station?

Reply:
it lets you charge a speed from gravity suit room even with door rando. I've already explained this in the discord. You break the pb blocks, go into the room, speed, spark. Whether you go in or start in it, it will be safe. Should also add it to this connection.
![image](https://user-images.githubusercontent.com/43147270/232233755-2f907b63-f683-418b-ae7f-47db91e21620.png)